static-build/CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ ExternalProject_Add(openssl
 #
 ExternalProject_Add(icu
     URL https://github.com/unicode-org/icu/releases/download/release-62-1/icu4c-62_1-src.tgz
+    URL_MD5 490ad9d920158e0314e10ba74ae9a150
```

Comment:
commit fbc04dfb uses variables with hash sums, could you make the same for ICU and libiconv?

Reply:
Fixed it myself and force-pushed.